service/controller/azure/v14/resource_set.go

```diff
@@ -335,6 +375,15 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 	return resourceSet, nil
 }
 
+func getClusterConfigFunc(obj interface{}) (v1alpha1.ClusterGuestConfig, error) {
```

Comment:
Let's drop the `Func` suffix from the actual function. It should be added only to variables and fields.